notebooks/official/migration/UJ6 Vertex SDK AutoML Text Classification.ipynb

```diff
@@ -8,7 +8,7 @@
       },
```

Comment:
<p>use the template version for REGION</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/904/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>